### PR TITLE
Main page layout

### PR DESCRIPTION
**Pull request by algolj**
Add Header, Footer, Not-found page, Space aside, header and filter-bar in board page.

### CODE REVIEW

File: src/app/auth/components/auth-button/auth-button.component.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
```
Comment (Wolf-Den1994) [inline src/app/auth/components/auth-button/auth-button.component.ts:4]: Is it important?
Maybe we should add this rule in eslint.json file?

File: src/app/core/components/footer/footer.component.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <a href="https://rs.school/angular/" target="_blank" class="footer__img-link"
+    ><img src="./assets/img/rs_school_js.svg" class="footer__img" alt="logo rs school"
+  /></a>
```
Comment (Wolf-Den1994) [inline src/app/core/components/footer/footer.component.html:4]: Why line breaks these, but not these?
```
<a href="https://rs.school/angular/" target="_blank" class="footer__img-link">
  <img src="./assets/img/rs_school_js.svg" class="footer__img" alt="logo rs school" />
</a>
```

File: src/app/core/components/header/header.component.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<header class="header">
+  <div class="header__main">
+    <a class="header__logo-link" routerLink="/"
+      ><img class="header__logo" src="https://ng.ant.design/assets/img/logo.svg" alt="app logo"
+    /></a>
```
Comment (Wolf-Den1994) [inline src/app/core/components/header/header.component.html:5]: I have the same question for this place

File: src/app/core/components/nav/nav.component.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { Component, Input } from '@angular/core';
+import { INavLink } from '../../interfaces/nav-link.interface';
+
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
```
Comment (Wolf-Den1994) [inline src/app/core/components/nav/nav.component.ts:5]: Maybe rule?

File: src/app/core/pages/not-found/not-found.component.html [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<div class="not-found">
+  <img class="not-found__picture" src="./assets/img/not-found.webp" alt="404 not found" />
+  <h2>404</h2>
+  <p>Oops, not found!</p>
+  <a class="not-found__link-to-freepik" href="https://www.freepik.com/vectors/mistake"
+    >Mistake vector created by pch.vector - www.freepik.com</a
+  >
```
Comment (Wolf-Den1994) [inline src/app/core/pages/not-found/not-found.component.html:7]: Why this, but not this?
```
<a class="not-found__link-to-freepik" href="https://www.freepik.com/vectors/mistake">
  Mistake vector created by pch.vector - www.freepik.com
</a>
```

**Review by Wolf-Den1994 — COMMENTED** [review]
it's usually, all good. we can discuss about my comment in the evening

**Review by Wolf-Den1994 — APPROVED** [review]

**Comment by PoMaKoM** [comment]
Согласен с @Wolf-Den1994, больше пометок нет

**Pull request merged**